pkgs/development/python-modules/json-schema-for-humans/default.nix

```diff
@@ -4,23 +4,15 @@
 
 buildPythonPackage rec {
   pname = "json-schema-for-humans";
-  version = "0.27.0";
+  version = "0.27.1";
 
   src = fetchFromGitHub {
     owner = "coveooss";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1r40i192z6aasil5vsgcgp5yvx392dhhqnfc2qxbxvpja6l3p6p2";
+    sha256 = "0d2a4a2lcqssr5g9rmc76f86nkqc9grixh507vzc9fi1h3gbi765";
   };
 
-  patches = [ (fetchurl {
-    url = "https://github.com/coveooss/json-schema-for-humans/commit/1fe2e2391da5a796204fd1889e4a11a53f83f7c9.patch";
-    sha256 = "0kpydpddlg0rib9snl8albhbrrs6d3ds292gpgpg7bdpqrwamdib";
-  }) (fetchurl {
```

Comment:
Missed that in the review but in the future please use fetchpatch instead of fetchurl.

Also please remove fetchurl from inputs.